app/package.json

```diff
@@ -14,6 +14,7 @@
     "postinstall": "yarn electron-rebuild"
   },
   "dependencies": {
-    "comit-sdk": "^0.17.0"
+    "comit-sdk": "^0.17.0",
+    "bitcoinjs-lib": "^5.1.10"
```

Comment:
Do you use this new lib somewhere?

Reply:
good point! this is actually not needed anymore, stayed in from a previous iteration :)
will remove.